architecture_docs/02_architecture_constraints.md

```diff
@@ -0,0 +1,24 @@
+# Architecture Constraints
+
+Loculus is developed under the following constraints:
+
+### Open Source Software
+
+We decided to develop Loculus under an open source license.
+The code is publicly available.
+
+Some aspects why we chose to develop Loculus as open source software:
+* to increase transparency of the project,
+* to allow others to contribute,
+* others are supposed to use the software - they should be able to see how it works.
```

Comment:
"they should be able to see how it works" sounds a bit redundant to the first point, what about something simpler like this?

```suggestion
* to allow others to use the software freely.
```